core/src/main/java/se/tre/freki/core/MetaClient.java

```diff
@@ -198,13 +199,18 @@ public void deleteAnnotation(final Annotation note) {
    */
   public ListenableFuture<Boolean> update(final LabelMeta meta) {
     return transform(getLabelMeta(meta.type(), meta.identifier()),
-        new AsyncFunction<LabelMeta, Boolean>() {
+        new AsyncFunction<Optional<LabelMeta>, Boolean>() {
```

Comment:
How does this work when a label is created without the extra `LabelMeta` information one tries to update the `LabelMeta` information after that?